plugins/provisioners/ansible/provisioner/base.rb

```diff
@@ -175,6 +176,15 @@ def prepare_common_command_arguments
           @command_arguments << "--start-at-task=#{config.start_at_task}" if config.start_at_task
         end
 
+        def get_inventory_command(inventory_path)
+          # print gathered version
+          if !@gathered_version_major.nil?  && !@gathered_version_minor.nil? && @gathered_version_major.to_i >= 2 && @gathered_version_minor.to_i >= 19
```

Comment:
you can drop the not nil checks here since the `.to_i` will force an integer for the check (with `nil.to_i` being `0`).